bob-grpc/proto/bob.proto

```diff
@@ -79,3 +86,9 @@ message BobError {
     int32 code = 1; // Error code in case of error
     string desc = 2; // Error desription
 }
+
+// Delete operation options
+message DeleteOptions {
+    bool force_node = 1; // Force operation to be served by node to which it comes
+    bool without_aliens = 2; // Do not remove data in aliens
```

Comment:
This flag will become meaningless after issue #533 will be implemented, because working through aliens will become the default behavior (just like `put` request, which does not have such flag). Probably, it is better to remove it in this PR?

Reply:
Sure, I'll remove it